pkg/csi_driver/node.go

```diff
@@ -69,7 +69,7 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 	}
 
 	// Validate volume attributes
-	attr := req.GetVolumeAttributes()
+	attr := req.GetVolumeContext()
 	if err := validateVolumeAttributes(attr); err != nil {
```

Comment:
Unfortunately we'll probably want to rename things like validateVolumeAttributes and validateSmbNodePublishSecrets to match the new API (eg, validVolumeContext and validateSmbSecrets, etc).

Reply:
Yes the cleanup can be done in future patches. This patch would be to just get the exisiting driver upto the latest spec.